api/sheets/get_variable.md

```diff
@@ -8,16 +8,27 @@ grand_parent: API
 
 # Get the value of a statistical variable at a given place and time.
 
-## `=DCGET(dcids, variable, date)`
+Given a list of [Place](https://datacommons.org/browser/Place) [DCIDs](/glossary.html), a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and optionally a date, get the measurements of the specified variable in the specified places with a date (if specified).
 
-Given a list of [Place](https://datacommons.org/browser/Place) DCIDs, a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and a date, get the measurements of that variable at those places on that date.
+## General information about this endpoint
 
-**Arguments**
-*   `dcids` - place DCIDs as a single value, a row, or a column
-*   `variable` - The StatisticalVariable to get
-*   `date` - The dates to observe in ISO 8601 format (e.g. 2017, "2017", "2017-12") or as a Google sheets [date value](https://support.google.com/docs/answer/3092969?hl=en). This can be a single value, a row, or a column.
+**Signature**: `=DCGET(dcids, variable, date)`
 
-`date` is optional. If it's not specified, the function returns the latest observation of the variable.
+**Required Arguments**:
+
+*   `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+
+*   `variable` - The StatisticalVariable whose measurement is sought.
+
+**Optional arguments**:
+
+*   `date` - The date or dates of interest.
+
+## Assembling the information you will need a request to the places within a place endpoint
+
+This endpoint requires the arguments `dcids` and `variable`. [DCIDs](/glossary.html) are unique node identifiers defined by Data Commons. Your query will need to specify the DCIDs for the parent places of interest. You are also required to specify the [statistical variable](/glossary.html) whose measurement you seek. Statistical variables are the metrics tracked by Data Commons.
+
+You may choose to specify the `date` argument. You may specify this argument as a single value, a row, or a column. All dates must be in ISO 8601 format (e.g. 2017, "2017", "2017-12") or as a Google sheets [date value](https://support.google.com/docs/answer/3092969?hl=en). If this argument is not specified, the API will return the latest observation of the variable.
 
 **Returns**
```

Comment:
I think this is important to add before the "Assembling" section above.

Reply:
I am not sure what is meant by this comment. I moved the third sentence of the second paragraph up; I thought about adding the ISO 8601 specification too, but I thought that might make the arguments section a bit unwieldy. Let me know your thoughts!

Reply:
I think this is clear!